src/components/Calls/CallNotification.js

```diff
@@ -15,15 +16,27 @@ import styles from './styles.scss';
 export const CallNotification = ({ callProvider, callerUsername, url, dispatch, time, rid } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined, time: undefined }) => {
 	const [show, setShow] = useState(true);
 
+	const callInNewTab = async () => {
+		const { token, room } = store.state;
+		const url = `${ Livechat.client.host }/meet/${ room._id }?token=${ token }`;
+		await dispatch({ ongoingCall: { callStatus: 'ongoingCallInNewTab', time: { time } }, incomingCallAlert: { show: false, callProvider } });
+		window.open(url);
+	};
+
 	const acceptClick = async () => {
 		setShow(!{ show });
 		await Livechat.updateCallStatus('inProgress', rid);
+
 		switch (callProvider) {
 			case constants.jitsiCallStartedMessageType: {
 				window.open(url);
 				break;
 			}
 			case constants.webrtcCallStartedMessageType: {
+				if (window.innerWidth <= 800 && window.innerHeight >= 630) {
```

Comment:
This seems to be a duplicate code being used in 2 places. So perhaps we can make a helper function in `src/components/helpers.js` named `isMobileDevice()` and call it from both these places